classes/core/String.inc.php

```diff
@@ -417,6 +417,7 @@ function mime_content_type($filename, $suggestedExtension = '') {
 			'sldx:application/zip' => 'application/vnd.openxmlformats-officedocument.presentationml.slide',
 			'docx:application/zip' => 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
 			'dotx:application/zip' => 'application/vnd.openxmlformats-officedocument.wordprocessingml.template',
+			'epub:application/zip' => 'application/epub+zip'
```

Comment:
I know this is super-picky, but can you put the comma back on the end of that line so that when the next change comes around it's just an add and not a modify+add?

With that, I think we're good to merge.